extensions/observability-devservices/testlibs/devresource-lgtm/src/main/java/io/quarkus/observability/devresource/lgtm/LgtmResource.java

```diff
@@ -61,9 +61,9 @@ public Map<String, String> doStart() {
 
         // set relevant properties for Quarkus extensions directly
         if (catalog != null && catalog.hasOpenTelemetry()) {
-            containerConfigs.put("quarkus.otel.exporter.otlp.traces.endpoint",
+            containerConfigs.put("quarkus.otel.exporter.otlp.endpoint",
                     String.format("http://%s:%s", host, container.getOtlpPort()));
-            containerConfigs.put("quarkus.otel.exporter.otlp.traces.protocol", "http/protobuf");
+            containerConfigs.put("quarkus.otel.exporter.otlp.protocol", "http/protobuf");
```

Comment:
The right protocol?

Reply:
yes, this will become the default soon and already works ok.